test/test_jit.py

```diff
@@ -15679,7 +15679,7 @@ def fn(string):
     def test_unicode_comments(self):
         @torch.jit.script
         def test(self, a):
-            # 🤷🤷🤷🤷
+            # shrug
```

Comment:
Well, this BE change actually renders test useless, as it supposed to test unicode comments... :/

Reply:
Sure - It seems that we should either tell flake8 to allow unicode or we should not have code with unicode in it.